nifi-nar-bundles/nifi-standard-services/nifi-record-serialization-services-bundle/nifi-record-serialization-services/src/main/java/org/apache/nifi/json/JsonTreeReader.java

```diff
@@ -66,6 +67,17 @@
     private volatile String dateFormat;
     private volatile String timeFormat;
     private volatile String timestampFormat;
+    private volatile String skipToNestedJsonField;
+
+    public static final PropertyDescriptor SKIP_TO_NESTED_JSON_FIELD = new PropertyDescriptor.Builder()
+            .name("skip-to-nested-json-field")
+            .displayName("Skip To Nested JSON Field")
+            .description("Skips forward to the given nested JSON field (array or object) and begins processing there. " +
+                    "If the field is not nested, the processing will proceed from the next available nested field.")
```

Comment:
This particular sentence is not quite clear. Should it say that if the Starting Field Name is not specified, then processing will begin at the first available element?

Reply:
I removed this part because the behavior got reverted to the original one, so it'll return an empty json in this case.